src/libstore/local.mk

```diff
@@ -44,9 +44,19 @@ libstore_CXXFLAGS += \
  -DNIX_MAN_DIR=\"$(mandir)\" \
  -DLSOF=\"$(lsof)\"
 
+ifeq ($(embedded_sandbox_shell),yes)
+libstore_CXXFLAGS += -DSANDBOX_SHELL=\"__embedded_sandbox_shell__\"
+
+$(d)/build/local-derivation-goal.cc: $(d)/embedded-sandbox-shell.gen.hh
+
+$(d)/embedded-sandbox-shell.gen.hh: $(sandbox_shell)
+	$(trace-gen) hexdump -v -e '1/1 "0x%x," "\n"' < $< > $@.tmp
```

Comment:
I'm not sure it's available in stdenv, but if it is, `xxd` has a `--include` option to directly produce a C-compatible header

Reply:
Unfortunately, `xxd` is part of `vim` which is a pretty big dependency to pull in...